pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -887,7 +904,7 @@ func (m *kubeGenericRuntimeManager) computeInitContainerActions(pod *v1.Pod, pod
 	// have been executed at some point in the past.  However, they could have been removed
 	// from the container runtime now, and if we proceed, it would appear as if they
 	// never ran and will re-execute improperly except for the restartable init containers.
-	podHasInitialized := hasAnyRegularContainerCreated(pod, podStatus)
+	podHasInitialized := isAnyRegularContainerRunning(pod, podStatus)
```

Comment:
let's move this method here to keep it inline so the diff with 1.27 will be more obvious.

Reply:
I mean do it inline instead of splitting into a separate method

Reply:
done